docs/robots/dingo/tutorials_dingo.mdx

```diff
@@ -4,7 +4,1023 @@ sidebar_position: 4
+
+:::note
+When simulating, you must leave Gazebo running. Closing Gazebo will prevent other tools, such as
+RViz (see below) from working correctly.
+:::
+
+:::note
+See also [Additional Simulation Worlds](#additional-sim).
+:::
+
+##### Simulation Configs
+
+Note that like Dingo itself, Dingo's simulator comes in multiple flavours called configs. A common
+one which you will need often is the `front_laser` config. If you close the Gazebo window, and then
+CTRL-C out of the terminal process, you can re-launch the simulator with a specific config.
+
+```bash
+roslaunch dingo_gazebo dingo_world.launch config:=front_laser
+```
+
+You should now see the simulator running with the simulated SICK LMS-111 laser present.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/dingo_images/dingo_gazebo_laser.png"
+      width="800"
+    />
+    <figcaption>Simulated dingo in the Race World with SICK LMS-111</figcaption>
+  </figure>
+</center>
+
+Gazebo not only simulates the physical presence of the laser scanner, it also provides simulated data
+which reflects the robot's surroundings in its simulated world. We will visualize the simulated laser
+scanner data shortly.
+
+### Interfacing with Dingo
+
+Both simulated and real Dingo robots expose the same ROS interface and can be interacted with 
+in the same way.
+
+:::note
+Please make sure that the desktop packages for Dingo are installed:
+
+```bash
+sudo apt-get install ros-noetic-dingo-desktop
+```
+:::
+
+#### Launching RViz
+
+The next tool we will encounter is [RViz](http://wiki.ros.org/rviz). Although superficially similar in
+appearance to Gazebo, RViz has a very different purpose. Unlike Gazebo, which shows the reality of the
+simulated world, rviz shows the robot's *perception* of its world, whether real or simulated. So while
+Gazebo won't be used with your real Dingo, RViz is used with both.
+
+You can use the following launch invocation to start RViz with a predefined configuration suitable for
+visualizing any standard Dingo config.
+
+```bash
+roslaunch dingo_viz view_robot.launch
+```
+
+You should see RViz appear.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/dingo_images/dingo_rviz.png"
+      width="800"
+    />
+    <figcaption>Dingo RViz</figcaption>
+  </figure>
+</center>
+
+The RViz display only shows what the robot knows about its world, which presently, is nothing. Because the
+robot doesn't yet know about the barriers which exist in its Gazebo world, they are not shown here.
+
+#### Driving with Interactive Controller
+
+RViz will also show Dingo's interactive markers around your Dingo's model. These will appear as a blue ring
+and red arrows. Depending on your robot, there will also be green arrows. If you don't see them in your RViz
+display, select the Interact tool from the top toolbar and they should appear. 
+
+Drag the red arrows in rviz to move in the linear x direction, and the blue circle to move in the angular z
```

Comment:
Change to 'RViz'.
Change 'X' and 'Z'  ?

Reply:
Updated